packages/table/examples/tableSortableExample.tsx

```diff
@@ -66,12 +68,12 @@ class TextSortableColumn extends AbstractSortableColumn {
 class RankSortableColumn extends AbstractSortableColumn {
     private static RANK_PATTERN = /([YOSKMJ])([0-9]+)(e|w)/i;
     private static TITLES: {[key: string]: number} = {
-        "Y": 0, // Yokozuna
-        "O": 1, // Ozeki
-        "S": 2, // Sekiwake
-        "K": 3, // Komusubi
-        "M": 4, // Maegashira
-        "J": 5, // Juryo
```

Comment:
FWIW I think inline disabling of the object-literal-sort-keys rule is sometimes necessary (more so than other rules)